src/errors/game.py

```diff
@@ -0,0 +1,2 @@
+class CellOccupied(BaseException):
```

Comment:
Why `BaseException`, and not just an Exception?

BaseException handle ALL exceptions at all, that you probably don't need.

Check this https://docs.python.org/3/library/exceptions.html#exception-hierarchy